Composer/packages/extensions/extension/src/hooks/useDialogApi.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { useShellApi } from './useShellApi';
+
+export const useDialogApi = () => {
+  const {
+    shellApi: { getDialog, saveDialog, createDialog },
+  } = useShellApi();
+
+  async function createDialogCompleted(): Promise<string | undefined> {
+    const newDialogId = await createDialog([]);
+    // TODO: @lei9444 be able to new dialog data here.
+    return newDialogId;
+  }
```

Comment:
@lei9444 could you help to implement the `createDialogComplteted` method here?
It requires the newly created dialog data is readable via `shellApi.getDialog`

Reply:
@yeze322 fix it in #2724